### PR TITLE
Add integration tests for ephemeral models rendered as empty operators

### PR DESCRIPTION
**Pull request by pankajkoti**
Adds integration coverage for #1133.

Adds a `customers -> ephemeral_customers -> ephemeral_customers_downstream` lineage to the `altered_jaffle_shop` test project and `DbtDag` tests (rendering via `dbt ls`) asserting that the ephemeral model renders as an `EmptyOperator` by default with the dependency chain preserved, and as a dbt run task when the option is disabled. Updates the node-count assertions for the two added models.

Regenerates the project's committed `manifest.json` (dbt 1.11.2, schema v12) for the new models; this also drops a stale multibyte model node whose `.sql` file had already been removed from disk.

> Stacked on #2759 — review and merge that first.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

**Comment by pankajkoti** [comment]
The PR is ready for review. Please do not merge this PR. It relies on https://github.com/astronomer/astronomer-cosmos/pull/2759 to be merged first.

**Review by tatiana — APPROVED** [review]
Thanks, @pankajkoti , these look great!

File: cosmos/airflow/graph.py [hunk 1]
```diff
@@ -466,6 +466,15 @@ def create_task_metadata(  # noqa: C901
...
 
+            if node.has_ephemeral_materialization and render_config.ephemeral_models_as_empty_operator:
+                # Ephemeral models are inlined as CTEs into downstream models and never written to the
+                # warehouse, so running them via a dbt operator is a no-op. Render them as empty operators
+                # to avoid wasted dbt invocations while keeping the node in the graph so that the dependency
+                # chain passing through it is preserved. EmptyOperator does not accept custom parameters
+                # (e.g. profile_args), so recreate the args keeping only the display name when present.
+                args = {"task_display_name": args["task_display_name"]} if "task_display_name" in args else {}
+                return TaskMetadata(id=task_id, operator_class=EMPTY_OPERATOR_CLASS_PATH, arguments=args)
+
```
Comment (pankajkoti) [inline cosmos/airflow/graph.py]:
```suggestion
```

File: cosmos/airflow/graph.py [hunk 2]
```diff
@@ -466,6 +466,15 @@ def create_task_metadata(  # noqa: C901
...
             )
 
+            if node.has_ephemeral_materialization and render_config.ephemeral_models_as_empty_operator:
+                # Ephemeral models are inlined as CTEs into downstream models and never written to the
+                # warehouse, so running them via a dbt operator is a no-op. Render them as empty operators
+                # to avoid wasted dbt invocations while keeping the node in the graph so that the dependency
+                # chain passing through it is preserved. EmptyOperator does not accept custom parameters
+                # (e.g. profile_args), so recreate the args keeping only the display name when present.
+                args = {"task_display_name": args["task_display_name"]} if "task_display_name" in args else {}
+                return TaskMetadata(id=task_id, operator_class=EMPTY_OPERATOR_CLASS_PATH, arguments=args)
```
[the remark this answers is not in the document]
Comment (pankajkoti) [reply cosmos/airflow/graph.py]: yes I identified this during my self-review and have removed it already.

File: tests/test_converter.py [hunk 3]
```diff
@@ -1519,3 +1523,60 @@ def test_telemetry_metadata_not_stored_when_disabled(mock_should_emit, mock_load
...
+            ),
+        ),
+        operator_args={"install_deps": True},
+    )
+
+
+@pytest.mark.integration
+def test_dbt_dag_renders_ephemeral_model_as_empty_operator():
+    """By default, an ephemeral model is rendered as an EmptyOperator, and the dependency chain that passes
+    through it (customers -> ephemeral_customers -> ephemeral_customers_downstream) is preserved."""
```
[the remark this answers is not in the document]
Comment (pankajkoti) [reply tests/test_converter.py:1559]: This is okay; we have unit tests covering the default check. https://github.com/astronomer/astronomer-cosmos/pull/2759/changes#diff-da24b355349b53e4c8047034767d008e4c2ce9ea6c8933a3acd50d0828f00528R281

Ignoring to avoid another CI run for this nitpick.
